composer.json

```diff
@@ -1,36 +1,30 @@
 {
   "name": "psr/log-util",
   "description": "Utilities for the psr/log package that backs the PSR-3 specification.",
-  "minimum-stability": "dev",
   "license": "MIT",
   "authors": [
     {
       "name": "PHP-FIG",
       "homepage": "https://www.php-fig.org/",
       "role": "Organisation"
-    },
-    {
-      "name": "Anton Ukhanev",
-      "email": "xedin.unknown@gmail.com",
-      "role": "Developer"
     }
   ],
   "require": {
     "php": "^5.3 | ^7.0 | ^8.0",
     "psr/log": "^1.1.4"
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.0 | ^5.0 | ^6.0 | ^7.0 | ^8.0 | ^9.0",
+    "phpunit/phpunit": "^8.0 | ^9.0",
     "squizlabs/php_codesniffer": "^3.6"
   },
   "autoload": {
     "psr-4": {
-      "Psr\\Log\\Util\\": "src"
+      "Psr\\Log\\": "src"
```

Comment:
This directory does not exist. Do you think I should add `src/.gitkeep` in order to avoid any potential issue with an autoloader?

Reply:
Removed the autoload from this branch.